### PR TITLE
 Swagger doc

### PR DESCRIPTION
**Pull request by soulshake**
## Release Playbook
- [ ] Rebase against master
- [ ] Code review
- [ ] Merge into master
- [ ] Verify changes at http://site-staging.convox.com
- [ ] Promote release on `site-production` in the `convox/production` Rack

* Add some missing parameters (in many places, we need to provide a `rack` parameter, otherwise the default rack is assumed)
* Add some missing methods (`/registries`, `/racks`)
* Add missing definitions (`organization`, `rack`...)
* Add example format for environment object
* Unify success description, and use it as the 200 `DELETE` response schema (instead of the schema of whatever object has been deleted, which is incorrect)

Note: the `/racks` method only partially works. We get `"error": "no response from server"` in the interactive API doc, but the displayed curl command works perfectly fine.

### CODE REVIEW

**Comment by soulshake** [comment]
[Update API doc content and examples](https://trello.com/c/hXG2iwZa/1384-update-api-doc-content-and-examples)

File: api/manifest.yml [hunk 1]
```diff
@@ -860,7 +1106,39 @@ paths:
...
 definitions:
+  rack:
+    properties:
+      id:
+        type: string
+      name:
+        type: string
+      host:
+        type: string
+      organization-id:
```
Comment (ddollar) [inline api/manifest.yml]: Pretty sure there should be no organization data in racks and no top level organization type.

Comment (soulshake) [inline api/manifest.yml]: An API call to `/racks` returns a list of dicts containing an `organization` key:

```
$ curl -X GET --header 'Content-Type: application/x-www-form-urlencoded' \
    --header 'Accept: application/json' \
    --header 'Authorization: Basic Y29udmredactedWUwOA==' \
    'https://cx.ngrok.io/racks' | jq .
[
  {
    "id": "54e149cc-8ef8-4a45-ab4e-1d9bfe543d99",
    "name": "pointless",
    "host": "pointless-775762807.us-west-2.elb.amazonaws.com",
    "organization-id": "775a6cfc-5caf-4299-ac23-25d200d3a1bf",
    "status": "converging",
    "version": "",
->  "organization": {
      "id": "775a6cfc-5caf-4299-ac23-25d200d3a1bf",
      "name": "personal",
      "DeveloperIds": [
        "3bd8e8cd-b93c-49c9-8956-74bb26ed1064"
      ]
    }
  },
[snip]
]
```

What do you mean by "top level organization type"? If you're referring to `'#/definitions/organization'` I'd like to understand what the problem is there. The way I understand the Swagger spec is that any nested dict-like object needs its own definition--that doesn't mean it corresponds to any API object. In the manifest, everything must have a `type` of either `string` or `array`; anything like a dict needs to have its own definition.

Comment (ddollar) [inline api/manifest.yml]: Hmm. Looks like that might be getting injected by Console. The Rack itself has no concept of organizations

**Pull request merged**